src/transformers/models/textnet/configuration_textnet.py

```diff
@@ -0,0 +1,136 @@
+# coding=utf-8
+# Copyright 2023 the Fast authors and HuggingFace Inc. team.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" TextNet model configuration"""
+from transformers import PretrainedConfig
+from transformers.utils import logging
+from transformers.utils.backbone_utils import BackboneConfigMixin, get_aligned_output_features_output_indices
+
+
+logger = logging.get_logger(__name__)
+
+TEXTNET_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "textnet-base": ("https://huggingface.co/Raghavan/textnet-base/blob/main/config.json"),
+}
+
+
+class TextNetConfig(BackboneConfigMixin, PretrainedConfig):
+    r"""
+        This is the configuration class to store the configuration of a [`TextNextModel`]. It is used to instantiate a
+    TextNext model according to the specified arguments, defining the model architecture. Instantiating a configuration
+    with the defaults will yield a similar configuration to that of the
+    [Raghavan/textnet-base](https://huggingface.co/Raghavan/textnet-base)Configuration objects inherit from
+    [`PretrainedConfig`] and can be used to control the model outputs.Read the documentation from [`PretrainedConfig`]
+    for more information.
+
+        Args:
+            stem_kernel_size (`int`, *optional*, defaults to 3):
+                The kernel size for the initial convolution layer.
+            stem_stride (`int`, *optional*, defaults to 2):
+                The stride for the initial convolution layer.
+            stem_num_channels (`int`, *optional*, defaults to 3):
+                The num of channels in input for the initial convolution layer.
+            stem_out_channels (`int`, *optional*, defaults to 64):
+                The num of channels in out for the initial convolution layer.
+            stem_act_func (`str`, *optional*, defaults to `"relu"`):
+                The activation function for the initial convolution layer.
+                                         [[3, 3], [1, 3], [3, 3], [3, 1]], [[3, 3], [3, 3], [3, 1], [1, 3]], [[3, 3], [3,
+                                         1], [1, 3], [3, 3]]]`):
+                The list of stagewise conv layer's kernel sizes.
+                                    [2, 1, 1, 1], [2, 1, 1, 1], [2, 1, 1, 1]]`):
+                The list of stagewise conv layer's kernel strides.
```

Comment:
Something funny has happened to the docstring here

Reply:
Yeah, I have reformatted.